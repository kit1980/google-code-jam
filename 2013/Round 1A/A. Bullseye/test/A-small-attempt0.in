1000
30 892
138 844
178 719
77 478
181 731
90 555
219 882
71 755
156 630
77 477
382 765
379 759
152 305
35 71
135 271
1 1000
190 381
190 442
327 655
327 960
77 477
77 476
77 478
77 579
35 783
35 782
35 784
35 845
399 799
399 905
115 705
115 704
115 706
115 864
191 770
191 769
191 771
191 944
88 925
88 924
88 926
88 972
28 750
28 749
28 751
28 836
25 228
25 227
25 229
25 242
265 531
265 787
171 690
171 689
171 691
171 765
58 762
58 761
58 763
58 810
384 769
384 864
154 939
154 938
154 940
154 946
175 351
175 662
35 225
35 224
35 226
35 249
7 330
7 329
7 331
7 372
98 603
98 602
98 604
98 699
97 394
97 393
97 395
97 535
59 369
59 368
59 370
59 455
301 603
301 797
189 379
189 473
11 611
11 610
11 612
11 644
450 901
450 986
453 907
453 951
275 551
275 842
127 255
127 375
323 647
323 820
98 398
98 397
98 399
98 533
45 606
45 605
45 607
45 650
125 765
125 764
125 766
125 798
309 619
309 710
206 413
206 541
283 567
283 719
408 817
408 823
78 318
78 317
78 319
78 462
425 851
425 874
352 705
352 830
285 571
285 656
180 361
180 664
314 629
314 638
314 629
314 991
50 101
205 411
205 489
17 684
17 683
17 685
17 686
105 868
105 867
105 869
105 875
85 525
85 524
85 526
85 605
368 737
368 839
104 422
104 421
104 423
104 487
74 785
74 784
74 786
74 848
49 309
49 308
49 310
49 372
8 162
95 386
95 385
95 387
95 480
254 509
254 806
45 495
45 494
45 496
45 590
76 153
76 264
267 535
267 901
102 205
102 387
14 944
14 943
14 945
14 983
13 328
13 327
13 329
13 344
15 490
15 489
15 491
15 516
61 381
61 380
61 382
61 496
366 733
366 970
22 897
22 896
22 898
22 929
64 262
64 261
64 263
64 394
49 654
49 653
49 655
49 719
214 429
214 673
99 820
99 819
99 821
99 899
28 57
28 70
80 326
80 325
80 327
80 351
100 615
100 614
100 616
100 803
272 545
272 783
55 595
55 594
55 596
55 663
27 552
27 551
27 553
27 607
140 855
140 854
140 856
140 957
45 495
45 494
45 496
45 553
160 646
160 645
160 647
160 706
270 541
270 647
70 588
70 587
70 589
70 638
300 601
300 747
23 275
23 274
23 276
23 332
252 505
252 646
55 595
55 594
55 596
55 724
336 673
336 679
4 270
4 269
4 271
4 298
112 225
112 440
94 189
94 318
23 828
23 827
23 829
23 922
176 710
176 709
176 711
176 817
110 446
110 445
110 447
110 527
236 473
236 854
211 850
211 849
211 851
211 912
96 796
96 795
96 797
96 950
47 749
47 748
47 750
47 871
40 348
40 347
40 349
40 393
20 135
20 134
20 136
20 143
99 402
99 401
99 403
99 603
419 839
419 880
454 909
454 979
350 701
350 722
261 523
261 994
213 609
22 548
22 550
22 615
368 747
190 621
200 805
200 807
200 927
34 663
34 665
34 734
71 754
71 756
71 873
249 601
1 252
1 254
1 259
222 497
105 867
105 869
105 874
400 917
280 786
185 745
185 747
185 1000
185 480
304 947
65 404
65 406
65 413
179 721
179 723
179 729
250 747
330 737
36 404
36 406
36 469
25 227
25 229
25 232
351 833
308 914
24 353
24 355
24 387
44 706
44 708
44 819
28 324
28 326
28 383
296 638
443 978
102 626
102 628
102 840
309 710
142 866
142 868
142 948
203 817
203 819
203 904
219 524
155 436
100 253
164 478
333 815
372 860
379 759
106 305
109 668
109 670
109 876
36 230
36 232
36 235
65 547
65 549
65 676
328 731
181 590
122 746
122 748
122 806
242 972
53 122
334 669
26 377
26 379
26 451
41 355
41 357
41 438
317 696
63 257
63 259
63 371
263 643
137 553
137 555
137 583
7 798
7 800
7 860
374 749
337 982
1 54
1 56
1 63
75 196
3 132
3 134
3 136
456 968
28 182
28 184
28 206
293 774
104 859
104 861
104 938
42 464
42 466
42 519
313 940
171 409
372 875
91 369
91 371
91 431
138 466
339 760
415 838
76 635
76 637
76 718
134 818
134 820
134 911
342 983
237 490
48 762
48 764
48 806
5 340
5 342
5 355
318 899
187 391
208 837
208 839
208 969
210 690
96 264
344 851
58 237
58 239
58 310
140 337
355 901
158 339
370 952
221 459
337 756
401 982
2 778
2 780
2 795
50 790
50 792
50 912
281 897
431 985
19 81
19 83
19 86
365 933
146 577
99 375
114 698
114 700
114 807
243 757
221 587
174 579
194 656
60 785
60 787
60 812
224 523
325 893
63 257
63 259
63 322
150 831
313 907
230 654
140 439
24 303
133 863
277 849
216 471
452 983
295 700
168 449
135 858
122 784
195 917
96 357
84 359
145 900
118 529
101 554
256 963
68 376
323 802
172 525
157 879
153 790
3 265
12 504
48 221
86 493
71 193
114 694
128 425
25 884
60 896
124 824
64 914
137 924
394 849
283 882
98 657
81 744
359 774
287 932
250 832
170 637
6 32
264 695
303 762
28 155
32 387
214 918
151 539
283 682
92 963
43 223
132 389
331 728
230 754
61 967
95 738
9 80
319 877
58 888
248 603
91 269
56 206
208 845
161 879
143 326
93 533
181 478
142 461
7 123
145 684
88 699
135 984
73 369
332 680
273 872
171 812
201 711
196 731
95 369
59 686
12 140
64 292
122 341
114 689
76 427
72 760
61 780
94 825
11 556
352 914
352 879
86 579
156 952
235 497
24 776
22 477
34 751
56 271
109 904
194 814
266 702
349 980
74 374
42 122
124 749
45 459
109 539
156 620
27 362
52 728
148 604
164 493
59 492
12 75
116 591
104 871
322 835
200 867
310 663
36 714
39 723
117 464
196 765
286 680
42 581
111 587
426 898
289 818
135 390
368 756
324 849
330 994
133 440
166 380
7 150
191 430
34 618
245 686
58 850
52 353
218 629
139 398
7 160
47 823
94 504
42 187
117 840
192 712
95 929
38 338
451 927
335 723
308 629
36 989
158 940
303 813
306 757
85 828
371 774
296 623
67 273
88 778
429 942
456 958
26 65
86 686
394 818
97 503
227 501
64 788
227 678
67 152
53 359
144 727
213 572
120 983
267 999
158 839
2 685
202 971
241 789
273 569
33 258
103 210
58 565
178 395
46 205
53 434
27 864
25 764
8 327
16 970
86 218
107 796
242 583
42 822
81 834
80 729
373 912
129 723
177 750
278 944
291 947
186 808
171 555
95 594
38 897
269 925
52 944
203 919
31 660
160 600
333 973
347 755
79 518
71 396
254 679
63 411
82 770
14 267
70 173
10 938
46 949
175 723
281 941
333 957
138 997
38 376
275 862
45 240
190 774
246 589
252 949
460 954
158 970
338 703
91 444
250 547
220 615
142 468
219 803
264 810
226 819
388 792
152 516
3 806
216 810
113 650
119 330
311 671
79 402
106 331
23 485
36 917
113 683
209 885
22 679
135 917
207 551
299 769
149 368
51 385
161 770
45 118
127 757
261 523
345 701
370 762
55 611
371 987
53 380
167 472
313 766
316 652
85 347
34 246
204 834
97 786
100 293
10 67
81 322
314 642
28 721
61 511
112 661
237 908
176 720
8 631
58 161
199 471
211 659
111 914
183 866
71 985
89 661
27 902
187 638
123 871
282 838
255 911
261 892
1 49
135 935
212 531
132 518
101 296
52 766
100 391
203 459
135 382
174 485
441 898
218 882
18 215
209 589
92 586
31 699
1 675
143 637
331 734
305 706
61 784
127 630
159 963
107 893
11 901
110 270
253 785
45 519
249 673
155 658
328 776
20 84
104 748
295 599
72 689
363 882
151 885
35 271
115 775
48 842
273 771
179 882
64 807
253 588
28 362
368 967
224 919
109 631
68 147
22 763
212 556
37 567
14 425
403 892
28 468
91 194
173 636
317 645
173 378
385 863
120 475
46 97
180 836
273 883
61 735
155 947
125 951
95 230
216 536
279 865
85 659
90 527
135 373
94 562
258 1000
4 141
237 803
178 822
271 755
3 145
191 481
347 835
223 451
6 776
93 932
26 886
261 670
79 414
207 558
164 369
58 525
183 371
430 969
289 892
208 953
94 855
355 910
266 582
42 394
2 761
108 703
84 484
5 959
252 819
248 571
163 839
210 434
235 839
275 577
54 845
410 882
225 626
185 557
190 536
24 465
65 484
348 872
432 931
273 966
258 728
260 542
120 244
48 490
60 430
215 587
205 950
96 998
103 764
205 594
254 738
16 859
400 831
42 358
16 1000
114 414
314 994
197 792
60 705
180 652
321 878
164 428
117 949
152 391
165 817
48 319
73 171
265 696
2 70
16 224
1 356
161 376
209 978
56 864
71 195
463 964
407 956
124 552
81 695
90 802
288 647
72 966
7 750
216 738
95 430
20 165
22 81
