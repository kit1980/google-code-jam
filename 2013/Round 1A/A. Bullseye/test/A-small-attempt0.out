Case #1: 11
Case #2: 3
Case #3: 2
Case #4: 3
Case #5: 2
Case #6: 3
Case #7: 2
Case #8: 5
Case #9: 2
Case #10: 3
Case #11: 1
Case #12: 1
Case #13: 1
Case #14: 1
Case #15: 1
Case #16: 22
Case #17: 1
Case #18: 1
Case #19: 1
Case #20: 1
Case #21: 3
Case #22: 2
Case #23: 3
Case #24: 3
Case #25: 9
Case #26: 8
Case #27: 9
Case #28: 9
Case #29: 1
Case #30: 1
Case #31: 3
Case #32: 2
Case #33: 3
Case #34: 3
Case #35: 2
Case #36: 1
Case #37: 2
Case #38: 2
Case #39: 5
Case #40: 4
Case #41: 5
Case #42: 5
Case #43: 10
Case #44: 9
Case #45: 10
Case #46: 10
Case #47: 4
Case #48: 3
Case #49: 4
Case #50: 4
Case #51: 1
Case #52: 1
Case #53: 2
Case #54: 1
Case #55: 2
Case #56: 2
Case #57: 6
Case #58: 5
Case #59: 6
Case #60: 6
Case #61: 1
Case #62: 1
Case #63: 3
Case #64: 2
Case #65: 3
Case #66: 3
Case #67: 1
Case #68: 1
Case #69: 3
Case #70: 2
Case #71: 3
Case #72: 3
Case #73: 10
Case #74: 9
Case #75: 10
Case #76: 10
Case #77: 3
Case #78: 2
Case #79: 3
Case #80: 3
Case #81: 2
Case #82: 1
Case #83: 2
Case #84: 2
Case #85: 3
Case #86: 2
Case #87: 3
Case #88: 3
Case #89: 1
Case #90: 1
Case #91: 1
Case #92: 1
Case #93: 13
Case #94: 12
Case #95: 13
Case #96: 13
Case #97: 1
Case #98: 1
Case #99: 1
Case #100: 1
Case #101: 1
Case #102: 1
Case #103: 1
Case #104: 1
Case #105: 1
Case #106: 1
Case #107: 2
Case #108: 1
Case #109: 2
Case #110: 2
Case #111: 6
Case #112: 5
Case #113: 6
Case #114: 6
Case #115: 3
Case #116: 2
Case #117: 3
Case #118: 3
Case #119: 1
Case #120: 1
Case #121: 1
Case #122: 1
Case #123: 1
Case #124: 1
Case #125: 1
Case #126: 1
Case #127: 2
Case #128: 1
Case #129: 2
Case #130: 2
Case #131: 1
Case #132: 1
Case #133: 1
Case #134: 1
Case #135: 1
Case #136: 1
Case #137: 1
Case #138: 1
Case #139: 1
Case #140: 1
Case #141: 1
Case #142: 1
Case #143: 1
Case #144: 1
Case #145: 1
Case #146: 12
Case #147: 11
Case #148: 12
Case #149: 12
Case #150: 4
Case #151: 3
Case #152: 4
Case #153: 4
Case #154: 3
Case #155: 2
Case #156: 3
Case #157: 3
Case #158: 1
Case #159: 1
Case #160: 2
Case #161: 1
Case #162: 2
Case #163: 2
Case #164: 5
Case #165: 4
Case #166: 5
Case #167: 5
Case #168: 3
Case #169: 2
Case #170: 3
Case #171: 3
Case #172: 6
Case #173: 2
Case #174: 1
Case #175: 2
Case #176: 2
Case #177: 1
Case #178: 1
Case #179: 5
Case #180: 4
Case #181: 5
Case #182: 5
Case #183: 1
Case #184: 1
Case #185: 1
Case #186: 1
Case #187: 1
Case #188: 1
Case #189: 16
Case #190: 15
Case #191: 16
Case #192: 16
Case #193: 8
Case #194: 7
Case #195: 8
Case #196: 8
Case #197: 10
Case #198: 9
Case #199: 10
Case #200: 10
Case #201: 3
Case #202: 2
Case #203: 3
Case #204: 3
Case #205: 1
Case #206: 1
Case #207: 13
Case #208: 12
Case #209: 13
Case #210: 13
Case #211: 2
Case #212: 1
Case #213: 2
Case #214: 2
Case #215: 6
Case #216: 5
Case #217: 6
Case #218: 6
Case #219: 1
Case #220: 1
Case #221: 4
Case #222: 3
Case #223: 4
Case #224: 4
Case #225: 1
Case #226: 1
Case #227: 2
Case #228: 1
Case #229: 2
Case #230: 2
Case #231: 3
Case #232: 2
Case #233: 3
Case #234: 3
Case #235: 1
Case #236: 1
Case #237: 5
Case #238: 4
Case #239: 5
Case #240: 5
Case #241: 8
Case #242: 7
Case #243: 8
Case #244: 8
Case #245: 3
Case #246: 2
Case #247: 3
Case #248: 3
Case #249: 5
Case #250: 4
Case #251: 5
Case #252: 5
Case #253: 2
Case #254: 1
Case #255: 2
Case #256: 2
Case #257: 1
Case #258: 1
Case #259: 4
Case #260: 3
Case #261: 4
Case #262: 4
Case #263: 1
Case #264: 1
Case #265: 5
Case #266: 4
Case #267: 5
Case #268: 5
Case #269: 1
Case #270: 1
Case #271: 5
Case #272: 4
Case #273: 5
Case #274: 5
Case #275: 1
Case #276: 1
Case #277: 10
Case #278: 9
Case #279: 10
Case #280: 10
Case #281: 1
Case #282: 1
Case #283: 1
Case #284: 1
Case #285: 12
Case #286: 11
Case #287: 12
Case #288: 12
Case #289: 2
Case #290: 1
Case #291: 2
Case #292: 2
Case #293: 2
Case #294: 1
Case #295: 2
Case #296: 2
Case #297: 1
Case #298: 1
Case #299: 2
Case #300: 1
Case #301: 2
Case #302: 2
Case #303: 4
Case #304: 3
Case #305: 4
Case #306: 4
Case #307: 7
Case #308: 6
Case #309: 7
Case #310: 7
Case #311: 4
Case #312: 3
Case #313: 4
Case #314: 4
Case #315: 3
Case #316: 2
Case #317: 3
Case #318: 3
Case #319: 2
Case #320: 1
Case #321: 2
Case #322: 2
Case #323: 1
Case #324: 1
Case #325: 1
Case #326: 1
Case #327: 1
Case #328: 1
Case #329: 1
Case #330: 1
Case #331: 1
Case #332: 8
Case #333: 9
Case #334: 9
Case #335: 1
Case #336: 1
Case #337: 1
Case #338: 2
Case #339: 2
Case #340: 7
Case #341: 8
Case #342: 8
Case #343: 4
Case #344: 5
Case #345: 5
Case #346: 1
Case #347: 10
Case #348: 11
Case #349: 11
Case #350: 1
Case #351: 3
Case #352: 4
Case #353: 4
Case #354: 1
Case #355: 1
Case #356: 1
Case #357: 2
Case #358: 2
Case #359: 1
Case #360: 1
Case #361: 2
Case #362: 3
Case #363: 3
Case #364: 1
Case #365: 2
Case #366: 2
Case #367: 1
Case #368: 1
Case #369: 4
Case #370: 5
Case #371: 5
Case #372: 3
Case #373: 4
Case #374: 4
Case #375: 1
Case #376: 1
Case #377: 5
Case #378: 6
Case #379: 6
Case #380: 6
Case #381: 7
Case #382: 7
Case #383: 4
Case #384: 5
Case #385: 5
Case #386: 1
Case #387: 1
Case #388: 2
Case #389: 3
Case #390: 3
Case #391: 1
Case #392: 2
Case #393: 3
Case #394: 3
Case #395: 1
Case #396: 2
Case #397: 2
Case #398: 1
Case #399: 1
Case #400: 1
Case #401: 1
Case #402: 1
Case #403: 1
Case #404: 1
Case #405: 1
Case #406: 2
Case #407: 3
Case #408: 3
Case #409: 2
Case #410: 3
Case #411: 3
Case #412: 3
Case #413: 4
Case #414: 4
Case #415: 1
Case #416: 1
Case #417: 2
Case #418: 3
Case #419: 3
Case #420: 1
Case #421: 1
Case #422: 1
Case #423: 5
Case #424: 6
Case #425: 6
Case #426: 3
Case #427: 4
Case #428: 4
Case #429: 1
Case #430: 1
Case #431: 2
Case #432: 2
Case #433: 1
Case #434: 1
Case #435: 2
Case #436: 2
Case #437: 16
Case #438: 17
Case #439: 17
Case #440: 1
Case #441: 1
Case #442: 4
Case #443: 5
Case #444: 5
Case #445: 1
Case #446: 6
Case #447: 7
Case #448: 7
Case #449: 1
Case #450: 2
Case #451: 3
Case #452: 3
Case #453: 1
Case #454: 3
Case #455: 4
Case #456: 4
Case #457: 4
Case #458: 5
Case #459: 5
Case #460: 1
Case #461: 1
Case #462: 1
Case #463: 1
Case #464: 2
Case #465: 2
Case #466: 1
Case #467: 1
Case #468: 1
Case #469: 3
Case #470: 4
Case #471: 4
Case #472: 2
Case #473: 3
Case #474: 3
Case #475: 1
Case #476: 1
Case #477: 6
Case #478: 7
Case #479: 7
Case #480: 10
Case #481: 11
Case #482: 11
Case #483: 1
Case #484: 1
Case #485: 1
Case #486: 2
Case #487: 2
Case #488: 1
Case #489: 1
Case #490: 1
Case #491: 1
Case #492: 2
Case #493: 2
Case #494: 1
Case #495: 1
Case #496: 1
Case #497: 1
Case #498: 1
Case #499: 1
Case #500: 1
Case #501: 18
Case #502: 19
Case #503: 19
Case #504: 6
Case #505: 7
Case #506: 7
Case #507: 1
Case #508: 1
Case #509: 1
Case #510: 2
Case #511: 2
Case #512: 1
Case #513: 1
Case #514: 1
Case #515: 2
Case #516: 3
Case #517: 3
Case #518: 1
Case #519: 1
Case #520: 1
Case #521: 1
Case #522: 5
Case #523: 6
Case #524: 6
Case #525: 1
Case #526: 1
Case #527: 1
Case #528: 2
Case #529: 2
Case #530: 2
Case #531: 1
Case #532: 1
Case #533: 1
Case #534: 5
Case #535: 3
Case #536: 1
Case #537: 1
Case #538: 1
Case #539: 1
Case #540: 1
Case #541: 3
Case #542: 3
Case #543: 2
Case #544: 1
Case #545: 2
Case #546: 3
Case #547: 2
Case #548: 2
Case #549: 1
Case #550: 2
Case #551: 1
Case #552: 1
Case #553: 2
Case #554: 2
Case #555: 10
Case #556: 11
Case #557: 2
Case #558: 2
Case #559: 1
Case #560: 2
Case #561: 1
Case #562: 12
Case #563: 6
Case #564: 3
Case #565: 6
Case #566: 3
Case #567: 1
Case #568: 1
Case #569: 3
Case #570: 4
Case #571: 1
Case #572: 1
Case #573: 1
Case #574: 1
Case #575: 2
Case #576: 1
Case #577: 1
Case #578: 2
Case #579: 5
Case #580: 2
Case #581: 1
Case #582: 1
Case #583: 4
Case #584: 2
Case #585: 1
Case #586: 1
Case #587: 1
Case #588: 7
Case #589: 3
Case #590: 3
Case #591: 1
Case #592: 6
Case #593: 1
Case #594: 1
Case #595: 1
Case #596: 2
Case #597: 2
Case #598: 1
Case #599: 2
Case #600: 1
Case #601: 1
Case #602: 5
Case #603: 2
Case #604: 3
Case #605: 3
Case #606: 2
Case #607: 1
Case #608: 1
Case #609: 2
Case #610: 1
Case #611: 1
Case #612: 1
Case #613: 5
Case #614: 4
Case #615: 2
Case #616: 1
Case #617: 2
Case #618: 2
Case #619: 4
Case #620: 5
Case #621: 4
Case #622: 12
Case #623: 1
Case #624: 1
Case #625: 3
Case #626: 3
Case #627: 1
Case #628: 11
Case #629: 8
Case #630: 8
Case #631: 2
Case #632: 4
Case #633: 2
Case #634: 1
Case #635: 1
Case #636: 2
Case #637: 1
Case #638: 2
Case #639: 4
Case #640: 2
Case #641: 1
Case #642: 5
Case #643: 6
Case #644: 2
Case #645: 1
Case #646: 3
Case #647: 2
Case #648: 2
Case #649: 4
Case #650: 1
Case #651: 2
Case #652: 1
Case #653: 8
Case #654: 7
Case #655: 1
Case #656: 1
Case #657: 1
Case #658: 6
Case #659: 2
Case #660: 1
Case #661: 1
Case #662: 1
Case #663: 1
Case #664: 1
Case #665: 1
Case #666: 1
Case #667: 1
Case #668: 6
Case #669: 1
Case #670: 7
Case #671: 1
Case #672: 6
Case #673: 3
Case #674: 1
Case #675: 1
Case #676: 6
Case #677: 7
Case #678: 2
Case #679: 2
Case #680: 3
Case #681: 1
Case #682: 4
Case #683: 4
Case #684: 1
Case #685: 1
Case #686: 1
Case #687: 10
Case #688: 2
Case #689: 1
Case #690: 1
Case #691: 4
Case #692: 1
Case #693: 1
Case #694: 1
Case #695: 4
Case #696: 1
Case #697: 1
Case #698: 1
Case #699: 3
Case #700: 1
Case #701: 2
Case #702: 1
Case #703: 5
Case #704: 1
Case #705: 1
Case #706: 3
Case #707: 2
Case #708: 1
Case #709: 3
Case #710: 1
Case #711: 2
Case #712: 17
Case #713: 2
Case #714: 1
Case #715: 1
Case #716: 3
Case #717: 1
Case #718: 4
Case #719: 1
Case #720: 2
Case #721: 3
Case #722: 11
Case #723: 10
Case #724: 9
Case #725: 15
Case #726: 1
Case #727: 3
Case #728: 1
Case #729: 8
Case #730: 4
Case #731: 4
Case #732: 1
Case #733: 2
Case #734: 2
Case #735: 1
Case #736: 1
Case #737: 2
Case #738: 1
Case #739: 3
Case #740: 9
Case #741: 1
Case #742: 7
Case #743: 2
Case #744: 8
Case #745: 1
Case #746: 1
Case #747: 1
Case #748: 3
Case #749: 2
Case #750: 1
Case #751: 3
Case #752: 4
Case #753: 6
Case #754: 1
Case #755: 17
Case #756: 8
Case #757: 2
Case #758: 1
Case #759: 1
Case #760: 3
Case #761: 4
Case #762: 1
Case #763: 2
Case #764: 2
Case #765: 1
Case #766: 1
Case #767: 1
Case #768: 3
Case #769: 1
Case #770: 2
Case #771: 1
Case #772: 1
Case #773: 1
Case #774: 1
Case #775: 1
Case #776: 1
Case #777: 1
Case #778: 1
Case #779: 18
Case #780: 1
Case #781: 2
Case #782: 1
Case #783: 1
Case #784: 2
Case #785: 1
Case #786: 7
Case #787: 10
Case #788: 2
Case #789: 2
Case #790: 10
Case #791: 3
Case #792: 1
Case #793: 1
Case #794: 1
Case #795: 3
Case #796: 2
Case #797: 1
Case #798: 2
Case #799: 1
Case #800: 1
Case #801: 1
Case #802: 5
Case #803: 1
Case #804: 3
Case #805: 1
Case #806: 1
Case #807: 1
Case #808: 2
Case #809: 3
Case #810: 2
Case #811: 3
Case #812: 1
Case #813: 2
Case #814: 1
Case #815: 1
Case #816: 9
Case #817: 3
Case #818: 2
Case #819: 1
Case #820: 2
Case #821: 14
Case #822: 1
Case #823: 1
Case #824: 1
Case #825: 3
Case #826: 2
Case #827: 6
Case #828: 3
Case #829: 11
Case #830: 1
Case #831: 3
Case #832: 1
Case #833: 1
Case #834: 1
Case #835: 4
Case #836: 3
Case #837: 1
Case #838: 1
Case #839: 1
Case #840: 6
Case #841: 1
Case #842: 1
Case #843: 1
Case #844: 1
Case #845: 1
Case #846: 2
Case #847: 4
Case #848: 1
Case #849: 3
Case #850: 8
Case #851: 18
Case #852: 2
Case #853: 1
Case #854: 1
Case #855: 5
Case #856: 2
Case #857: 2
Case #858: 4
Case #859: 16
Case #860: 1
Case #861: 1
Case #862: 5
Case #863: 1
Case #864: 2
Case #865: 1
Case #866: 1
Case #867: 3
Case #868: 1
Case #869: 4
Case #870: 1
Case #871: 2
Case #872: 3
Case #873: 3
Case #874: 7
Case #875: 1
Case #876: 2
Case #877: 5
Case #878: 1
Case #879: 5
Case #880: 1
Case #881: 2
Case #882: 2
Case #883: 1
Case #884: 11
Case #885: 1
Case #886: 6
Case #887: 9
Case #888: 1
Case #889: 6
Case #890: 1
Case #891: 1
Case #892: 1
Case #893: 1
Case #894: 1
Case #895: 1
Case #896: 1
Case #897: 2
Case #898: 1
Case #899: 5
Case #900: 3
Case #901: 3
Case #902: 1
Case #903: 1
Case #904: 1
Case #905: 3
Case #906: 2
Case #907: 1
Case #908: 2
Case #909: 1
Case #910: 6
Case #911: 1
Case #912: 2
Case #913: 1
Case #914: 7
Case #915: 1
Case #916: 1
Case #917: 1
Case #918: 17
Case #919: 4
Case #920: 11
Case #921: 1
Case #922: 2
Case #923: 1
Case #924: 1
Case #925: 4
Case #926: 1
Case #927: 1
Case #928: 1
Case #929: 2
Case #930: 4
Case #931: 1
Case #932: 1
Case #933: 4
Case #934: 18
Case #935: 3
Case #936: 2
Case #937: 19
Case #938: 1
Case #939: 1
Case #940: 2
Case #941: 1
Case #942: 1
Case #943: 1
Case #944: 6
Case #945: 1
Case #946: 1
Case #947: 1
Case #948: 1
Case #949: 7
Case #950: 3
Case #951: 1
Case #952: 1
Case #953: 1
Case #954: 1
Case #955: 1
Case #956: 1
Case #957: 4
Case #958: 3
Case #959: 1
Case #960: 2
Case #961: 4
Case #962: 3
Case #963: 1
Case #964: 1
Case #965: 14
Case #966: 1
Case #967: 3
Case #968: 15
Case #969: 1
Case #970: 1
Case #971: 1
Case #972: 5
Case #973: 1
Case #974: 1
Case #975: 1
Case #976: 3
Case #977: 1
Case #978: 2
Case #979: 3
Case #980: 1
Case #981: 1
Case #982: 5
Case #983: 5
Case #984: 13
Case #985: 1
Case #986: 2
Case #987: 6
Case #988: 1
Case #989: 1
Case #990: 1
Case #991: 2
Case #992: 4
Case #993: 4
Case #994: 1
Case #995: 6
Case #996: 16
Case #997: 1
Case #998: 2
Case #999: 3
Case #1000: 1
